### PR TITLE
Update README, because nothing is ever simple

### PR DESCRIPTION
**Pull request by Burgestrand**
This is an irk of mine. No ill was meant when these words were written, but I feel using the word "simply" is a bit crass.

## To do

- [x] I have read the [contributing guidelines](https://github.com/varvet/pundit/contribute).
- [x] I have added relevant tests.
- [x] I have adjusted relevant documentation.
- [x] I have made sure the individual commits are meaningful.
- [x] I have added relevant lines to the CHANGELOG.

PS: Thank you for contributing to Pundit ❤️

### CODE REVIEW

**Comment by Burgestrand** [comment]
@dgmstuart I'm not sure if this is the improvement I intend it to be. wdyt, is it better or worse?

**Review by dgmstuart — APPROVED** [review]
I have some comments but mostly this feels like an improvement.

File: README.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ can pick up any classes in the new `app/policies/` directory.
 ## Policies
 
 Pundit is focused around the notion of policy classes. We suggest that you put
-these classes in `app/policies`. This is a simple example that allows updating
-a post if the user is an admin, or if the post is unpublished:
```
[the remark this answers is not in the document]
Comment (Burgestrand) [reply README.md:53]: Probably!

File: README.md [hunk 2]
```diff
@@ -419,20 +419,13 @@ class Post
 end
 ```
 
-## Just plain old Ruby
```
[the remark this answers is not in the document]
Comment (Burgestrand) [reply README.md:422]: Yeah, I see here it's used for emphasis. I don't care enough to make new commit to change it, but I agree it could have stayed!

**Pull request merged**